community/concepts/deep-then-wide.md

```diff
@@ -1,30 +1,51 @@
 
- - do we want to go "wide" (adoption by lots of projects) or 
- - "deep" (intensive usage by a few projects)?
-
-If you are familiar with depth vs. breadth when it comes to searching, this idea
-is similar to that. In the next sections, we'll talk about each option, and
-what it would mean for development and the future of the project.
+ - do we want to go "wide" (adoption by lots of projects) or
+ - go "deep" (intensive usage by a few projects)?
 
 #### What does it mean to go wide?
 
-Going "wide" is easier. SourceCred can run on any GitHub repository, so we have a path for engagement with the entire GitHub community. We could focus on getting more and more projects to try running SourceCred, and then convert them to regular usage. If we went down this route, we would focus on features that make the existing cred scores more salient and engaging — for example, making it easy to generate "weekly reports" showing the new cred being earned by contributors.
+Going "wide" means encouraging lots of communities to start using SourceCred.
+Since SourceCred can run on any GitHub repository or Discourse forum, we could
+roll cred out to many different projects and groups. We could optimize for the
+number of projects using SourceCred, and focus on features that make the scores
+more salient or engaging — for example, making it easy to generate "weekly
+reports" showing the new cred being earned by contributors.
 
-As an added bonus to going wide, progress is much easier to measure, as it’s easier to quantify the number of projects using SourceCred, but hard to quantify the amount of value that SourceCred is adding to any particular project.
+Going wide is tempting because progress is very easy to measure, and the project
+would get plenty of buzz which could bring more contributors and opportunities.
 
 #### What does it mean to go deep?
 
-Going "deep" is more challenging. It means finding a way to make SourceCred deeply and durably valuable to individual communities — not as a passing interest, but as something that gets a lot of community engagement, and becomes a part of how that community operates and thinks about itself.
+Going deep means that instead of focusing on finding many projects that will
+use SourceCred, we'll instead focus on making SourceCred work incredibly well
+for a few communities. In part, this means paying a lot of attention to the edge
+cases, and spots where SourceCred isn't working as intended, and trying to fix
+those issues.
+
+In many ways, going deep is more challenging than going wide. We could go wide
+by marketing SourceCred and building buzz, but improving the core quality requires
+a lot of experimentation with algorithms and community practices.
+
+#### Going Deep First
+
+We've decided to go deep before going wide. That means our priority is to prove
+SourceCred can do a really great job at attributing cred with early-adopter
+communities, before we try to scale for broader adoption.
```

Comment:
For the mentions of cred, maybe link to the other document (or should we add this when they both are merged?) It should be a relative link like `[cred](cred.md)`.